www/content/claviers/arsenik/index.md

```diff
@@ -0,0 +1,251 @@
+la réduire doucement au fil de la progression.
+
+### Homerow-mods
+
+De même que la couche NavNum est associée à [Espace]{.kbd}, les modifieurs
+[Ctrl]{.kbd}, [Alt]{.kbd}, [Cmd]{.kbd} sont associés aux touches de repos des
+deux mains : [N]{.kbd} [E]{.kbd} [S]{.kbd} pour la main gauche, [R]{.kbd}
+[T]{.kbd} [I]{.kbd} pour la main droite.
+
+![](hrm.svg)
+
+Là encore, la touche se comporte comme un *tap* tant qu’elle n’est pas maintenue
+au moins 300 ms ; on accepte donc un délai avant d’activer un modifieur.
+
+Plus ce délai est long et moins on est susceptible de déclencher des modifieurs
+par erreur quand on ne relève pas les doigts suffisamment vite. [Shift]{.kbd} et
+[AltGr]{.kbd} n’étant pas affectées par ce délai, on peut le rallonger sans que
+ça ne gêne la vitesse ou le confort de saisie.
+
+On se fait assez rapidement aux layer-taps sous les pouces, mais les
+homerow-mods sont plus longs à acquérir : on a tendance à ne pas lever les
+doigts assez vite, ce qui risque de déclencher des modifieurs par erreur.
+
+Certain·e·s ne s’y font jamais ; d’autres contournent le problème avec des
+[combo-mods][] ou des [callum-mods][]. Mais le jeu en vaut vraiment la
+chandelle : le gain de confort est phénoménal ! En étant patient et en utilisant
+des temporisations assez longues au début (300 ms voire 400 ms), on finit par
+s’y faire et ça devient une évidence.
+
+Pour creuser le sujet, [ce billet][precondition] fait référence de nos jours.
+
+
+Claviers de <i lang="en">laptop</i>
+--------------------------------------------------------------------------------
+
+### Kanata
+
+[kanata][] est un logiciel libre et multiplate-formes permettant de redéfinir le
+comportement des touches du clavier. Il permet notamment l’application de
+l’angle-mod, la définition de layers, la configuration de layer-taps et
+homerow-mods — et plus généralement, de faire une grande partie de ce que font
+les <i lang="en">firmwares</i> de claviers programmables comme QMK et ZMK.
+
+Une configuration kanata personnalisable est proposée ici : [arsenik.zip][]
+
+Cette configuration a pour but de faire découvrir Arsenik étape par étape :
+
+1. <i lang="en">angle-mod</i>
+2. <i lang="en">layer-taps</i>
+3. <i lang="en">homerow-mods</i>
+
+Par défaut, seul l’angle-mod est appliqué ; à vous d’activer les layer-taps et
+les homerow-mods à votre rythme.
+
+### 1. <i lang="en">Angle-mod</i>
+
+On applique un [angle-mod][], c’est-à-dire une permutation circulaire sur les 6
+touches de gauche de la rangée inférieure, afin de mieux respecter l’angle des
+poignets et de se rapprocher d’une disposition ortholinéaire :
+
+![Ergo‑L sur un clavier ISO](./ergol_iso.svg)
+
+![Ergo‑L avec l’angle-mod](./ergol_isoa.svg)
+
+Les barres d’espace des laptops ayant généralement une largeur de 5u, cet
+angle-mod symétrise aussi le placement des deux autres touches de pouces, qui se
+retrouvent dans l’alignement des majeurs.
+
```

Comment:
Nitpick / question : Do you think the `hints` theme is enough to explain that you still use the same fingers for `zxcvb` ? We could add a sentence to *really* drive the point home, but it may be redundant.

Reply:
I kinda assumed the visual hint would be enough, yes, as it’s all we use for the Duck Typist page. 

I agree we might want to reconsider this on a global basis if it turns out to be unclear as it affects (or will affect) : 
- the upcoming installation page
- the current Duck Typist
- the Arsenik page.

So far I’ve pointed to the Colemak-DH explanation page and hoped for the best.